cmd/hub/main.go

```diff
@@ -12,9 +13,20 @@ import (
 	"github.com/sonm-io/core/insonmnia/hub"
 )
 
+var (
+	configPath = flag.String("config", "hub.yaml", "Path to hub config file")
+)
+
 func main() {
+	flag.Parse()
+
+	conf, err := hub.NewConfig(*configPath)
+	if err != nil {
+		panic("Cannot load config file: " + err.Error())
```

Comment:
It should be not 0. However some OS specifies their own exit codes for a good style. For example FreeBSD recommends to exit with EX_USAGE status.

For this moment I'd rather exit with 1 for the sake of not make things complex.